src/main/java/com/googlecode/lanterna/screen/AbstractScreen.java

```diff
@@ -191,7 +191,7 @@ public TextCharacter getBackCharacter(int column, int row) {
 
     @Override
     public void refresh() throws IOException {
-        refresh(RefreshType.AUTOMATIC);
+        refresh(RefreshType.DELTA);
```

Comment:
Rather, I think we should leave it at automatic and have the code handling automatic mode prefer DELTA.